src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml

```diff
@@ -2,11 +2,12 @@
              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:controls="clr-namespace:Dynamo.Controls"
```

Comment:
Is this line needed? Just curious since I see no reference of controls, did I miss anything?

Reply:
No, we can remove it. I updated the references.